tests/tests_requirements.txt

```diff
@@ -0,0 +1 @@
+coveralls==1.1
```

Comment:
header and description for this file would be great too. sorry for being picky, it just helps long term and it's relatively low cost :)